canonical-schema.json

```
@@ -113,6 +114,12 @@
           , "type"       : "string"
           },
 
+      "uuid":
+          { "description": "A unique identifier"
```

```suggestion
          { "description": "A universally unique identifier in the v4 format"
```

`1` would also be a unique identifier.